src/ui/settings.ts

```diff
@@ -59,7 +59,10 @@ export class SettingTab extends PluginSettingTab {
       this.addRuleToTab(tabTitle, rule);
     }
 
-    this.createTabAndContent('Custom', navEl, settingsEl, (el: HTMLElement, tabName: string) => this.generateCustomCommandSettings(tabName, el));
+    this.createTabAndContent('Custom', navEl, settingsEl, (el: HTMLElement, tabName: string) => {
+      this.generateCustomCommandSettings(tabName, el);
+      this.generateCustomRegexReplacementSettings(tabName, el);
```

Comment:
These probably need to be added into their own divs so the search for them will not work in a wonky manner.